solidity/contracts/libs/Call.sol

```diff
@@ -9,43 +9,78 @@ library CallLib {
         bytes data;
     }
 
-    function _multicall(address to, bytes[] memory calls) internal {
+    struct CallWithValue {
```

Comment:
Why have a separate type?

Reply:
Ah for IQS, ofc. Maybe `Call` and `StaticCall`?

Reply:
`StaticCall` is pretty easy to conflate with `address.staticcall`

Reply:
Exactly, I'm proposing that if you're using the struct without the value field that it's only used for static calls

Reply:
i.e.
```
struct StaticCall {
  address to;
  bytes data;
}

struct Call {
  address to;
  bytes data;
  uint256 value;
}

    function _staticcall(StaticCall memory call)
        internal
        view
        returns (bytes memory);

    function _staticcall(StaticCall memory call, bytes memory cb)
        internal
        view
        returns (bytes memory);

    function multicall(StaticCall[] memory calls) internal view returns (bytes[] memory);
    function multicall(StaticCall[] memory calls, bytes[] memory cbs) internal view returns (bytes[] memory);

    
     function _call(Call memory call) internal returns (bytes memory);
     function _call(Call memory call, bytes memory cb) internal returns (bytes memory);
    function multicall(Call[] memory calls) internal returns (bytes[] memory);
    function multicall(Call[] memory calls, bytes[] memory cbs) internal returns (bytes[] memory);
```